automation_tools/satellite6/capsule.py

```diff
@@ -393,7 +611,7 @@ def setup_capsules(path):
         with settings(host_string=capsule.host_string):
             register_capsule()
             put(local_path=cert_path)
-            capsule_installer(*oauth_info)
+            capsule_installer(capsule.hostname, cert_path, *oauth_info)
 
     # Start content synchronization in all registered capsules
```

Comment:
We can now do syncing from the WEBUI too, so I feel we should have a arg/flag to enable ot disable it.